Pinta.Tools/Tools/TextTool.cs

```diff
@@ -405,7 +405,8 @@ protected override void OnActivated (Document? document)
 
 		protected override void OnCommit (Document? document)
 		{
-			StopEditing (false);
+			imContext.FocusOut();
```

Comment:
Should this happen inside any call to `StopEditing()` rather than only in `OnCommit()`? (I don't know what the correct answer is)

Reply:
I am also not sure about it.
I only add FocusOut before the StopEditing, to make sure that imContext will leave focus while OnCommit.